src/test/java/uk/gov/dhsc/htbhf/steps/AddressLookupSteps.java

```diff
@@ -73,6 +73,14 @@ public void listOfAddressesAreShown() {
         verifyAddressOptionsText(POSTCODE, addressOptions);
     }
 
+    @Then("^the list of addresses starts with a count of matching addresses")
+    public void listOfAddressesIncludesCountOfMatchingAddresses() {
+        List<WebElement> addressOptions = getPages().getSelectAddressPage().getAllAddressOptions();
+        assertThat(addressOptions).hasSizeGreaterThan(1);
+        assertThat(addressOptions.get(0).getText()).endsWith("addresses found");
```

Comment:
Should we not be asserting the actual size here?

Reply:
I figured that's not what we're testing - there's another method that tests we get the correct number of addresses. Here we're only concerned with the first entry (so in theory it could be used with different postcodes without change).

Reply:
Seeing as the Scenario calls `I am shown a list of addresses` above and we assert that it has two options, I think we should specifically assert that there are 3 here, if we change the number of addresses then that other method is going to break anyway, so we'd may as well be specific that there are 2 selectable addresses and 1 option which is not selectable. Either that or get rid of `assertThat(addressOptions).hasSizeGreaterThan(1);` completely.

Reply:
My thinking is that this is a separate assertion (`the list of addresses starts with a count of matching addresses`) - it shouldn't need to know how any addresses there are, just that the list starts with the number of addresses (which requires that there be at least one option). However it's not worth holding up the PR for so if anyone still thinks I should change it I will.

Reply:
Ok, fair point well made. In which case I think my issue is with `assertThat(addressOptions).hasSizeGreaterThan(1);` because if the postcode match returns no results then this will be incorrect. So maybe we just need to remove that line? Another thing we should probably consider is whether we validate the number, we could easily at this point get the number from that text and make sure that the remaining number of entries matches that. Actually, now I've said that, I think that is the way to go, does that make sense?